boskos/janitor/compute_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/pborman/uuid"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/compute/v1"
+)
+
+func TestDelete(t *testing.T) {
```

Comment:
generally we want unit tests here to be hermetic and not rely on actually creating or deleting real resources - this is more of an integration test, which is fine, but which we probably don't want to run automatically.

if we want to unit test this, we might need to use fakes, or possibly try using something like cloud.google.com/go/httpreplay or cloud.google.com/go/rpcreplay.

Reply:
Added a "manual" tag to the test, so it won't be executed automatically.
We agreed that unit-tests aren't too useful for this functionalit.